index.html

```diff
@@ -215,7 +215,50 @@ <h2 class="holder-title">
     <h3 class="last-title-container">
         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
     </h3>
-    </div>
+    <form class="form-block">
```

Comment:
- [ ] Kindly make sure you use the formspree service to collect user data on your web page as specified in the project requirement. :smile: 
`hint: You may consider checking` [here](https://formspree.io/html/) `how to use formspree`. 
`If it still not clear to you, you may consider watching this video` [here](https://www.youtube.com/watch?v=H_MUJ--iAcc&t=18s)